workshops/static/import_from_url.js

```diff
@@ -8,6 +8,7 @@ function import_from_url(url) {
     $("#id_start").val(data.start);
     $("#id_end").val(data.end);
     $("#id_reg_key").val(data.reg_key);
+    $("#id_tags").val(data.tags);
```

Comment:
If I understand correctly, `data.tags` contains a list of tags like 'DC', 'TTT', 'SWC', or 'PyData', right?

I don't think I'm able to discover if a workshop is DC or SWC type. `data.tags` should be empty in this case.